boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -208,8 +210,16 @@ openapi-generate:
 			-h /dev/null \
 			-r "-"
 
+.PHONY: manifests
+manifests:
+# Only use kustomize to template out manifests if the path config/default exists
+ifneq (,$(wildcard config/default))
+	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
+	$(KUSTOMIZE) build config/default | $(YQ) -s '"deploy/" + .metadata.name + "." + .kind + ".yaml"'
```

Comment:
Can you add an else with an `@echo bla bla`? Otherwise this will break as there will be no command with the target.

Reply:
Oh interesting, I did try this with AVO and it worked, but I do like the idea of a log anyway to encourage people to swap to kustomize and maybe I have a newer version of make or something